186-render-annotations-in-corpus-texts

ctot-nondef opened the pull request.

simar0at committed:
Enable new window handling, add css for annotations

simar0at committed:
Add function that converts JSON strings to object values

simar0at committed:
Add a way to write down complex objects in data attributes

simar0at committed:
Merge remote-tracking branch 'origin/develop' into 186-render-annotations-in-corpus-texts

simar0at committed:
Add label to the parameters from element text

Not only create a window title.

simar0at committed:
Add label to corpus text parameters

simar0at committed:
Label is now part of the parameters

ctot-nondef committed:
feat: window type for corpus texts as JSON

ctot-nondef committed:
update branch

ctot-nondef committed:
feat: corpus view with annotations from json

ctot-nondef committed:
feat: types

ctot-nondef committed:
fix: format

ctot-nondef committed:
fix: removed audioplayer for now

ctot-nondef committed:
feat: replace old corpustext window

ctot-nondef committed:
fix: remove superfluous props

ctot-nondef committed:
fix: remove Dockerfile

ctot-nondef committed:
fix: refactor zod import

ctot-nondef committed:
fix: format

ctot-nondef committed:
fix: refactor zod imports

ctot-nondef committed:
fix: force node >=22.12.0 for vite 7+

simar0at committed:
An annotationBlock has either one u or an array of us

Render each u in a table row regardless.

simar0at committed:
Declare u or us in annotationBlock

simar0at committed:
In Tunocent there are no translations in spanGrp

simar0at committed:
Clona the object into param to avoid circular references for now

charlymo commented:
Und TUNICO! Ich habe mir das gerade angeschaut. Das sieht eigentlich ziemlich unproblematisch aus.

simar0at committed:
Add properties that should be configurable for demos to the window state

simar0at committed:
Simplify UI and make more UI state persistent

Persistent state:
* Query UI visible
* Manual or Builder

Watch for window parameter changes and update URL.
Remove "extended" UI and pagination (missing component).

simar0at committed:
Update the component to use the Vue 3.4+ defineModel macro

simar0at committed:
Make the text input for query templates part of the window state

simar0at committed:
Add a state field for the template query user input

simar0at committed:
Add a button that opens a dictionary entry

simar0at committed:
Merge branch '254-dictionary-query-widget-improve-usability' into 186-render-annotations-in-corpus-texts

simar0at committed:
Fix: this forced access breaks now in SHAWI

simar0at committed:
Add all the missing new parameters for a DictQuery window

I thought I made more optional. Needs review.

simar0at committed:
Hack so we get a "compact view" of the dict entry

ctot-nondef committed:
Merge branch 'develop' into 186-render-annotations-in-corpus-texts

ctot-nondef committed:
fix: limit engines to node 22.x

ctot-nondef committed:
fix: limit engines to node 22.x

ctot-nondef committed:
fix: limit engines to node 22.x

ctot-nondef committed:
fix: limit engines to node 22.x

ctot-nondef committed:
fix: limit engines to node 22.x

ctot-nondef committed:
fix: update pnpm to 9.15.9

ctot-nondef committed:
fix: revert to pnpm 9.10.0

ctot-nondef committed:
Merge branch 'develop' into 186-render-annotations-in-corpus-texts

# Conflicts:
#	components/corpus-query-window-content.vue
#	components/corpus-text-window-content.vue
#	components/dict-query-window-content.vue
#	components/geo-map-toolbar.vue
#	components/geo-map-window-content.vue
#	components/geojson-map-legend.vue
#	components/geojson-map-toolbar.vue
#	components/geojson-map-window-content.vue
#	composables/use-biblio-tei-query.ts
#	composables/use-dicts-dict.ts
#	composables/use-dicts-entries.ts
#	composables/use-explore-samples-results.ts
#	composables/use-feature-by-id.ts
#	composables/use-geo-markers.ts
#	composables/use-profile-by-id.ts
#	composables/use-sample-text-by-id.ts
#	composables/use-text-by-id.ts
#	pnpm-lock.yaml
#	stores/use-dict-store.ts
#	types/global.d.ts

ctot-nondef committed:
fix: adjusted to new format

ctot-nondef committed:
fix: render phr(ases)

ctot-nondef committed:
fix: remove manual typings

ctot-nondef committed:
fix: freeze node version

ctot-nondef committed:
feat: try node 24

simar0at committed:
We need to update builder (gliderlabs/herokuish) for node 24

simar0at committed:
Set the builder images explicitly

ctot-nondef committed:
fix: some updates

ctot-nondef committed:
fix: remove phr element

simar0at committed:
Fix: hiding dvStats removed the UI header and the UI to (un)collapse

ctot-nondef committed:
fix: accomodate for multiple us within one divs array

ctot-nondef committed:
fix: spacing fix

ctot-nondef committed:
fix: speakerID

ctot-nondef committed:
fix: move utterance rendering to separate function

ctot-nondef committed:
fix: Dict link in separate function

ctot-nondef committed:
fix: include ID title

ctot-nondef committed:
Merge branch '254-dictionary-query-widget-improve-usability' into 186-render-annotations-in-corpus-texts

ctot-nondef committed:
fix: changes for 254 merge

ctot-nondef committed:
fix: prelim deduplication for dictquery
